db/migrate/20231115154524_add_parents_key_to_accounts.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddParentsKeyToAccounts < ActiveRecord::Migration[6.1]
+  def change
+    add_column :accounts, :parents_key, :integer
```

Comment:
```suggestion
    add_column :accounts, :parental_key, :string
```

It should be string because leading 0 matters